pytest_romancal/stpsf_plugin.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers", "stpsf: mark test as requiring stpsf data to run"
+    )
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--stpsf", action="store_true", default=False, help="run stpsf tests"
```

Comment:
This drops `--webbpsf` as an understood option for pytest runs. This will break the regression tests which provide that option:
https://github.com/spacetelescope/RegressionTests/blob/4643904b13e9d11df09e77b0c55ebb0de8275252/.github/workflows/romancal.yml#L108
However I think this PR will already require regression test changes as those only download webbpsf (not stpsf) data.

Reply:
There are no regtests that mark `webbpsf`. The whole of the repo has been scrubbed and there are no longer any strings of `webbpsf` remaining. It has been erased 🪹

Reply:
I think Brett is worried about the RegressionTests repo, i.e., https://github.com/spacetelescope/RegressionTests/blob/4643904b13e9d11df09e77b0c55ebb0de8275252/.github/workflows/romancal.yml#L6 .  That will need another PR.

Reply:
https://github.com/spacetelescope/RegressionTests/pull/194 is ready to go once this PR is merged.